wqflask/wqflask/templates/base.html

```diff
@@ -252,7 +252,6 @@
             })
     </script>
     <script src="{{ url_for('js', filename='jquery-cookie/jquery.cookie.js') }}" type="text/javascript"></script>
-    <script type="text/javascript" src="/static/new/js_external/json2.js"></script>
```

Comment:
**issue:** @Alexanderlacuna how come we have no replacement for this?

Reply:
I see that this files exists here:  wqflask/wqflask/static/new/js_external/json2.js

Reply:
@BonfaceKilz  Json is now included in the js language so I don't think we need the library.